demo/Makefile

```
@@ -21,20 +21,23 @@ manifests/s3-secret.yaml:
 	read -rp 'Enter your IONOS S3 secret key: ' SECRET_KEY && export SECRET_KEY && \
 	yq '.stringData.accessKey = env(ACCESS_KEY) | .stringData.secretKey = env(SECRET_KEY)' example-manifests/s3-secret.yaml > manifests/s3-secret.yaml
 
+# Maybe we should pin down the versions of these dependencies
 dependencies:
 	if [ -z "$(DEMO_LOGGING)" ]; \
 	then stackablectl operator install commons secret; \
 	else stackablectl stack install logging; fi
```

```suggestion
dependencies:
	if [ -z "$(DEMO_LOGGING)" ]; \
	then stackablectl operator install commons=23.4.1 secret=23.4.1 ; \
	else stackablectl stack install logging; fi
```

I've pinned the operator install, but the Stacks aren't versioned.